TypeCobol.Test/Parser/Programs/Cobol85/ExecSql/ExecSqlWithSelectDotStar.rdzSQL.txt

```diff
@@ -0,0 +1,16 @@
+﻿--- Sql Statements ---
+line 6: SelectStatement
+- FullSelect = FullSelect
+  - SubSelect = SubSelect
+    - SelectClause = SelectClause
+      - SelectionModifier = <NULL>
+      - Selections = [
+        - Selections[0] = TableOrViewAllColumnsSelection
+          - TableOrViewOrCorrelationName = TableViewCorrelationName
+            - Name = i IN i IN d
```

Comment:
At first I thought the order was modified because of Cobol but in fact the SymbolReferences we created so far are reversed. Let's imagine a 3-part name like `DBMS.schema.table`. The 'head' part of the corresponding SymbolReference is 'table' and the 'tail' part is 'DBMS.schema'. So you have to change the order when creating `QualifiedSymbolReference `and then update the result files.